src/action_server/actions/answer_question.py

```diff
@@ -0,0 +1,76 @@
+from action import Action
+
+import rospy
+from robocup_knowledge import load_knowledge
+
+class AnswerQuestion(Action):
+    def __init__(self):
+        Action.__init__(self)
+
+    def _configure(self, robot, config):
+        if not hasattr(robot, "speech"):
```

Comment:
I think you can make this a nice loop and specify a list of required skills in the constructor. 
Also, think about the number of arms required for an action